ikarus/solver/nonlinearsolver/newtonraphson.hh

```diff
@@ -163,4 +226,8 @@ auto makeNewtonRaphson(const NLO& nonLinearOperator, LS&& linearSolver = {}, UF&
                                                       std::move(updateFunction));
```

Comment:
`makeNewtonRaphson` can be removed similar to `makeNewtonRaphsonWithSubsidiaryFunction` -> Also fix the documentation?